kouncil-frontend/src/app/navbar/navbar.component.ts

```diff
@@ -3,6 +3,12 @@ import {SearchService} from 'app/search.service';
 import {Router} from '@angular/router';
 import {Observable} from 'rxjs';
 import {HttpClient} from '@angular/common/http';
+import {Globals} from '../globals';
+
+export class Server {
```

Comment:
to na bank tu powinno być zdefiniowane? ;)

Reply:
początkowo tutaj to była, ale zapomniałem przenieść. poprawię.